include/GoogleSync/GoogleSync.php

```diff
@@ -874,30 +1044,15 @@ public function updateGoogleCalendarEvent($event_local, $event_remote)
         $endDateTime->setTimeZone($this->timezone);
         $event_remote->setEnd($endDateTime);
 
-        // We pull the existing extendedProperties, and change our values
-        // That way we don't mess with anything else that's using other values.
-        $extendedProperties = $event_remote->getExtendedProperties();
-
-        if (!empty($extendedProperties)) {
-            $private = $extendedProperties->getPrivate();
-        } else {
-            $extendedProperties = new Google_Service_Calendar_EventExtendedProperties;
-            $private = array();
-        }
-
-        $private['suitecrm_id'] = $event_local->id;
-        $private['suitecrm_type'] = $event_local->module_name;
-
-        $extendedProperties->setPrivate($private);
+        $extendedProperties = $this->returnExtendedProperties($event_remote, $event_local);
         $event_remote->setExtendedProperties($extendedProperties);
 
         // Copy over popup reminders
         $event_local_id = $event_local->id;
         $reminders_local = BeanFactory::getBean('Reminders')->get_full_list(
-            "",
-            "reminders.related_event_module = 'Meetings'" .
-            " AND reminders.related_event_module_id = '$event_local_id'" .
-            " AND popup = '1'"
+                "", "reminders.related_event_module = 'Meetings'" .
+                " AND reminders.related_event_module_id = '$event_local_id'" .
```

Comment:
possible sql injection: $event_local_id should be escaped and validated before concatenated into sql query.

Reply:
Done